packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -225,163 +332,220 @@ @interface FLTMarkersController ()
                               mapView:(GMSMapView *)mapView
                             registrar:(NSObject<FlutterPluginRegistrar> *)registrar {
-  self = [super init];
-  if (self) {
-    _methodChannel = methodChannel;
-    _mapView = mapView;
-    _markerIdentifierToController = [[NSMutableDictionary alloc] init];
-    _registrar = registrar;
-  }
-  return self;
+    self = [super init];
+    if (self) {
+        _methodChannel = methodChannel;
+        _mapView = mapView;
+        _markerIdentifierToController = [[NSMutableDictionary alloc] init];
+        _registrar = registrar;
+        _emptyClusterMarker = [self baseClusterMarker];
+        _fontPath = [self loadFont];
+    }
+    return self;
+}
+
+
+void CFSafeRelease(CFTypeRef cf) {
+    if (cf != NULL) {
+        CFRelease(cf);
+    }
+}
+
+-(NSString *)loadFont {
+    NSBundle *frameworkBundle = [NSBundle bundleForClass:self.classForCoder];
+    NSURL *bundleURL = [[frameworkBundle resourceURL] URLByAppendingPathComponent:@"CozyFonts.bundle"];
+    NSBundle *bundle = [NSBundle  bundleWithURL:bundleURL];
+    NSURL *fontURL = [bundle URLForResource:@"oatmealpro2_semibold" withExtension:@"otf"];
+    NSData *inData = [NSData dataWithContentsOfURL:fontURL];
+    CFErrorRef error;
+    CGDataProviderRef provider = CGDataProviderCreateWithCFData((CFDataRef)inData);
+    CGFontRef font = CGFontCreateWithDataProvider(provider);
+    if (!CTFontManagerRegisterGraphicsFont(font, &error)) {
+        CFStringRef errorDescription = CFErrorCopyDescription(error);
+        NSLog(@"Failed to load font: %@", errorDescription);
+        CFRelease(errorDescription);
+    }
+    NSString *fontName = (__bridge NSString *)CGFontCopyPostScriptName(font);
+    CFSafeRelease(font);
+    CFSafeRelease(provider);
+    return fontName;
+}
+
+
+-(UIImage *)baseClusterMarker {
+    CGFloat size = 66;
+    CGSize canvas = CGSizeMake(size, size);
+    UIGraphicsBeginImageContext(canvas);
+    UIGraphicsImageRenderer *renderer = [[UIGraphicsImageRenderer alloc] initWithSize: canvas];
+    UIImage *image = [renderer imageWithActions:^(UIGraphicsImageRendererContext * _Nonnull rendererContext) {
+        CGContextSetAlpha(rendererContext.CGContext, 0.05);
+        CGContextSetFillColorWithColor(rendererContext.CGContext, UIColor.grayColor.CGColor);
+        CGContextAddEllipseInRect(rendererContext.CGContext, CGRectMake(0, 0, size, size));
+        CGContextDrawPath(rendererContext.CGContext, 3);
+        CGContextSetAlpha(rendererContext.CGContext, 1.0);
+        CGContextSetFillColorWithColor(rendererContext.CGContext, UIColor.whiteColor.CGColor);
+        CGContextSetStrokeColorWithColor(rendererContext.CGContext, UIColor.clearColor.CGColor);
+        CGContextSetLineWidth(rendererContext.CGContext, 10);
+        CGContextAddEllipseInRect(rendererContext.CGContext, CGRectMake(1, 1, size - 2, size - 2));
+        CGContextDrawPath(rendererContext.CGContext, 3);
```

Comment:
Assuming that what we wanted here was `Fill Rect`, the constant you could use here is `kCGPathFillStroke` 😉